src/main/scala/scalismo/package.scala

```diff
@@ -38,6 +38,10 @@ package object scalismo {
       gc.SetScheduleTime(60, TimeUnit.SECONDS)
       gc.Start()
 
+      // we need to stop the garbage collection when the JVM shutdowns.
+      // Otherwise it prevents the JVM from shutting down correctly.
+      Runtime.getRuntime().addShutdownHook(new Thread { gc.Stop() })
```

Comment:
PS: From https://docs.oracle.com/javase/7/docs/api/java/lang/Runtime.html#addShutdownHook(java.lang.Thread)

"A shutdown hook is simply an initialized but unstarted thread. When the virtual machine begins its shutdown sequence it will start all registered shutdown hooks in some unspecified order and let them run concurrently"